src/wifi.rs

```diff
@@ -799,6 +799,49 @@ impl<'d> WifiDriver<'d> {
         })
     }
 
+    /// Get information of AP which the ESP32 station is associated with.
+    /// Usefull to get the current signal strength of the AP, in a simple way.
+    pub fn get_ap_info(&mut self) -> Result<AccessPointInfo, EspError> {
```

Comment:
should i add an additional check is_sta_connected() or just relight on the EspError ( does the job here)